tests/paddle/test_operators.py

```diff
@@ -114,7 +114,7 @@ def convert_float_to_uint16(float_list):
     new_output = []
     for x in np.nditer(float_list):
         new_output.append(np.uint16(copy_bits_from_float_to_uint16(x)))
-    new_output = np.reshape(new_output, float_list.shape).view(np.uint16)
+    new_output = np.reshape(new_output, float_list.shape).reshape(np.uint16)
```

Comment:
[`np.ndarray.reshape`](https://numpy.org/doc/2.0/reference/generated/numpy.ndarray.reshape.html) does not support dtype conversion:
```suggestion
    new_output = np.reshape(new_output, float_list.shape).view(np.uint16)
```